rhtconsulting/rhc-ose
# Updated tools to support separating out etcd for storage

#### What does this PR do?

The tools now support separating out etcd for storage. By default, starting with this PR, /var/lib/openshift does no longer get its own slice, while /var/lib/etcd does. This can all be configured by changing the configuration parameters to non-zero for which ones to configure as part of installation. 
#### How should this be manually tested?

Run a full install, using the command below:

```
./osc-provision --num-nodes=2 --key=<key_name> --ose-version=3.1
```

After installation completes, verify that /var/lib/etcd has been separated out, and that data is being written to it - i.e. using the following commands:

```
df | grep etcd <<-- should return a separate mount point for /var/lib/etcd
```

```
ls /var/lib/etcd <<-- should return a "member" directory
ls /var/lib/etcd/member <<-- should return some directories like "snap" and "wal"
```
#### Is there a relevant Issue open for this?
#87
#### Who would you like to review this?

/cc @etsauer
